Source/Mosa.Utility.Launcher/Builder.cs

```diff
@@ -373,6 +383,15 @@ private void CreateISOImageWithSyslinux()
 				File.WriteAllBytes(Path.Combine(isoDirectory, "mboot.c32"), GetResource(@"syslinux\3.72", "mboot.c32"));
 			}
 
+			var dir = Path.Combine(Environment.CurrentDirectory.Replace("bin", string.Empty), LauncherSettings.IncludePath);
```

Comment:
Same.

Reply:
Yes but then I don't know how to do it :/

Reply:
Pass in LauncherSettings.IncludePath rather than dir (and only if not null)

Reply:
but IncludePath isn't the full path, only part of.

Reply:
Okay. Combine with current path.

Reply:
Which is what I'm doing right now, so should I just keep it as is?

Reply:
Still need to remove the code that removes “bin”. That’s a path specific thing other project might not want.

Reply:
But I don't know how to get the current MOSA path other than doing that.

Reply:
Just remove “ .Replace("bin", string.Empty)” and check for null or empty string

Reply:
But then that would just get the location of the bin folder, wouldn't it?

Reply:
I've done better path handling, do you think it's good now?